spec/omniauth/strategies/realme_spec.rb

```diff
@@ -19,6 +19,10 @@
       legacy_rails_session_behaviour_enabled: legacy_rails_session_behaviour_enabled
     }
   end
+  let(:authenticity_token) do
```

Comment:
I wonder if you should include a comment just to describe why you need this authenticity stuff. It might be a bit confusing in the future.